_posts/2019-07-30-microfrontends-holidaycheck-part2.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title:  "Micro Frontends at HolidayCheck (part 2) - Speed"
+categories: microfrontend javascript 
+author_name: Sebastian Bruckner
+author_url : /author/sebastianbruckner
+author_avatar: sebruck
+read_time : 10
+feature_image: posts/2019-06-08-browsertools-2/ben-harritt-75854-unsplash.jpg
+---
+
+In the [part 1](/post/2019/06/28/microfrontends-holidaycheck-part1) of this series, I describe why and how we do micro frontends at HolidayCheck. One of the key
+aspects I described was the focus on speed. Our first micro frontend, the header, may not slow down the pages
+on which it is embedded. In this second article I will describe what we have done & learned along the way.
+The things described are not specific to micro frontends but can be applied in a variety of web apps,
+however our first micro frontend is the first project where we had a that strong focus on speed.
+
+## Performance Budgets
+
+Before we started planning the technical setup, we sat together and defined performance budgets for the header.
+You can see a performance budget as inverse speed limits for you application, so it defines the maximum amount of slowness you are willing 
+ to accept. Performance budgets can either focus on the input like the maximum amount of JavaScript you want to deliver or the outcome like
+[First Contentful Paint (FCP)](https://developers.google.com/web/tools/lighthouse/audits/first-contentful-paint) or [Time To Interactive (TTI)](https://developers.google.com/web/tools/lighthouse/audits/time-to-interactive). 
+We decided to use input metrics, since they are much easier to measure and influence by a micro frontend.
+
+We have defined the following budget:
```

Comment:
Is it possible to explain how did you come to these numbers? like `max. 32 kB minified`